app/Http/Controllers/Apis/Protected/Summit/Factories/Registration/PromoCodesValidationRulesFactory.php

```diff
@@ -154,7 +154,9 @@ public static function buildForUpdate(array $payload = []): array
             'valid_until_date'     => 'nullable|required_with:valid_since_date|date_format:U|after:valid_since_date',
             'allowed_ticket_types' => 'sometimes|int_array',
             'badge_features'       => 'sometimes|int_array',
-            'badge_features_apply_to_all_tix_retroactively' => 'sometimes|boolean'
+            'badge_features_apply_to_all_tix_retroactively' => 'sometimes|boolean',
+            'tags'                 => 'sometimes|string_array',
+            'track_id'             => 'nullable|required_with:tags|integer',
```

Comment:
this is not correct
u could add any tag to promo code
tags does not depends on track id
please remove this restriction

Reply:
fixed